.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,43 @@
+name: Build & Publish NPM Package
+
+on:
+  push:
+    tags:
+      - '**'
+
+concurrency:
+  group: "release"
+  cancel-in-progress: true
+
+jobs:
+  js_bindings:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+
+      - name: Get the version
+        id: version_info
+        run: echo ::set-output name=SOURCE_TAG::${GITHUB_REF#refs/tags/}
+
+      - name: Update version in package.json
+        working-directory: ${{ github.workspace }}/js-bindings
```

Comment:
Needs to be updated for this repo - I think we might actually just be able to drop for this, since working directory i think is just the root of the repo